addons/payment_stripe/tests/test_stripe.py

```diff
@@ -81,7 +88,8 @@ def test_30_stripe_form_management(self):
         self.assertEqual(tx.state, 'done', 'Stripe: validation did not put tx into done state')
         self.assertEqual(tx.acquirer_reference, stripe_post_data.get('id'), 'Stripe: validation did not update tx id')
 
-    def test_add_available_payment_method_types(self):
+    def test_add_available_payment_method_types_local_enabled(self):
+        self.stripe.payment_icon_ids = [(4, icon.id) for icon in self.all_icons]
```

Comment:
You can merge the setUp and this loop with the command `(6, 0, id_list)`.
As per the doc:
> (6, 0, ids)
replaces all existing records in the set by the ids list, equivalent to using the command 5 followed by a command 4 for each id in ids.

Reply:
Done